src/main/java/com/google/sps/servlets/NewProjectServlet.java

```diff
@@ -23,21 +23,25 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet responsible for creating new tasks. */
-@WebServlet("/new-task")
-public class NewTaskServlet extends HttpServlet {
+/** Servlet responsible for creating new projects. */
+@WebServlet("/post-project")
+public class NewProjectServlet extends HttpServlet {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String title = request.getParameter("title");
+    String description = request.getParameter("description");
     long timestamp = System.currentTimeMillis();
+    String giturl = request.getParameter("giturl");
 
-    Entity taskEntity = new Entity("Task");
-    taskEntity.setProperty("title", title);
-    taskEntity.setProperty("timestamp", timestamp);
+    Entity projectEntity = new Entity("Project");
```

Comment:
Would be helpful if we have some sort of doc or at least a comment at the top of the file showing the schema for each entity. This way collaborators don't have to dig through the code to find the schema.